src/foam/core/JSON.js

```diff
@@ -523,6 +612,30 @@ foam.LIB({
       useShortNames: false,
       strict: false,
       propertyPredicate: function(o, p) { return ! p.storageTransient; }
+    }),
+
+    // JSON (not FON): Short, but exclude network-transient properties.
+    NetworkJSON: foam.json.JSONParserStringifier.create({
+      pretty: false,
+      formatDatesAsNumbers: true,
+      outputDefaultValues: false,
+      // TODO: No deserialization support for shortnames yet.
+      //      useShortNames: true,
+      useShortNames: false,
+      strict: false,
```

Comment:
So in the base JSON Outputer.  strict:true was meant to output proper JSON.  Did you test if that works for you?

If we take all the flags of the standard Network config but also turn on strict: true, can we then parse it with JSON.parse?